aiohttp_sse/__init__.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         self.headers["Connection"] = "keep-alive"
         self.headers["X-Accel-Buffering"] = "no"
 
-        self._ping_interval: int = self.DEFAULT_PING_INTERVAL
+        self._ping_interval: Union[int, float] = self.DEFAULT_PING_INTERVAL
```

Comment:
Int is compatible with float:
```suggestion
        self._ping_interval: float = self.DEFAULT_PING_INTERVAL
```
Same for all the others.

Reply:
Wow, this is new info for me! Thanks. Fixed